pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go

```diff
@@ -199,6 +216,13 @@ func encodeTuned(tuned *tunedv1.Tuned) ([]byte, error) {
 	return tunedEncoded, err
 }
 
+func encodeMachineConfig(machineConfig *mcfgv1.MachineConfig) ([]byte, error) {
+	scheme := runtime.NewScheme()
+	mcfgv1.AddToScheme(scheme)
```

Comment:
the scheme is not supposed to change during the lifetime of the server, so do we need to do this for every invocation?